draft-ietf-rtcweb-jsep.xml

```diff
@@ -3541,16 +3541,21 @@ candidate:1 1 UDP 1694498815 192.0.2.33 10000 typ host
             RTCP component, and begin or continue muxing RTCP over the
             RTP component, as specified in
             <xref target="RFC5761" />, Section 5.1.3. Otherwise,
-            transmit RTCP over the RTCP component; if no RTCP component
-            exists, because RTCP mux was previously enabled, this MUST
-            result in an error.</t>
+            prepare to transmit RTCP over the RTCP component; if no RTCP
+            component exists, because RTCP mux was previously enabled, this
+            MUST result in an error.</t>
 
             <t>If the media section has reduced-size RTCP enabled,
             configure the RTCP transmission for this media section to
             use reduced-size RTCP, as specified in
             <xref target="RFC5506" />.</t>
 
-            <t>[TODO: enable appropriate rtcp-fb mechanisms]</t>
+            <t>Enable the RTCP feedback mechanisms referenced in the media
+            section for the specified media formats. Specifically,
+            begin or continue sending the requested feedback types and
+            reacting to received feedback, as specified in
```

Comment:
My intent was that begin or continue would act upon both 'sending' and 'reacting'. I could go either way.